src/gc/gcinterface.ee.h

```diff
@@ -221,17 +221,17 @@ class IGCToCLR {
     virtual
     uint32_t GetActiveSyncBlockCount() = 0;
 
-    // Queries whether or not the given thread has preemptive GC disabled.
+    // Queries whether or not the current thread has preemptive GC disabled.
     virtual
-    bool IsPreemptiveGCDisabled(Thread * pThread) = 0;
+    bool IsPreemptiveGCDisabled() = 0;
```

Comment:
Nit: I always found the double negations like this very hard to reason about. Would it make sense to call this `IsPreemptiveGCModeEnable` ?

Reply:
Maybe we can replace all of the PreemptiveGCDisabled in the whole runtime and GC in a separate PR. I've also found the naming to make thinking about the state difficult. There are 312 occurences total.